src/odemis/gui/cont/features.py

```diff
@@ -74,6 +75,41 @@ def _on_btn_go_to_feature(self, _):
         self._main_data_model.stage.moveAbs({'x': pos[0], 'y': pos[1]})
         self._main_data_model.focus.moveAbs({'z': pos[2]})
 
+    def _enable_feature_ctrls(self, enable):
+        self._panel.cmb_feature_status.Enable(enable)
```

Comment:
Please add the doctring:
       
       """
       enables/disables the feature controls

        :param (boolean) enable: enable/disable the feature controls 
        """